sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/README.md

```diff
@@ -1,13 +1,227 @@
+
+[Source code](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor) | [Package (PyPi)][pypi] | [API reference documentation][api_docs] | [Product documentation][product_docs] | [Samples](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/samples) | [Changelog](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/CHANGELOG.md)
+
+> **NOTE**: This is the preview for the next major version of the [`opentelemetry-azure-monitor-python`](https://github.com/microsoft/opentelemetry-azure-monitor-python).
 
 ## Getting started
 
+### Install the package
+
+Install the Microsoft Opentelemetry Exporter Azure Monitor client library for Python with [pip][pip]:
+
+```Bash
+pip install microsoft-opentelemetry-exporter-azuremonitor --pre
+```
+
+### Prerequisites: 
+To use this package, you must have:
+* Azure subscription - [Create a free account][azure_sub]
+* Azure Monitor - [How to use application insights][application_insights_namespace]
+* Opentelemetry SDK - [Opentelemtry SDK for Python][ot_sdk_python]
+* Python 3.5 or later - [Install Python][python]
+
+### Authenticate the client
+
+Interaction with Azure monitor exporter starts with an instance of the `AzureMonitorSpanExporter` class. You will need an **instrumentation_key** or a **connection_string** to instantiate the object.
+The priority of which value takes on the instrumentation key is:
+* Key from explicitly passed in connection string
+* Key from explicitly passed in instrumentation key
+* Key from connection string in environment variable
+* Key from instrumentation key in environment variable
+Please find the samples linked below for demonstration as to how to authenticate via either approach.
+
+#### [Create Exporter from connection string][sample_authenticate_client_connstr]
+
+```Python
+from microsoft.opentelemetry.exporter.azuremonitor import AzureMonitorSpanExporter
+exporter = AzureMonitorSpanExporter(
+    connection_string = os.environ["AZURE_MONITOR_CONNECTION_STRING"]
+)
+```
+
+#### Create Exporter using the instrumentation key:
+
+```Python
+from microsoft.opentelemetry.exporter.azuremonitor import AzureMonitorSpanExporter
+exporter = AzureMonitorSpanExporter(
+    instrumentation_key = os.environ["AZURE_MONITOR_INSTRUMENTATION_KEY"]
+)
+```
+
 ## Key concepts
 
+Some of the key concepts for the Azure monitor exporter include:
+
+* [Opentelemetry][opentelemtry_spec]: Opentelemetry is a tool to instrument, generate, collect, and export telemetry data (metrics, logs, and traces) for analysis in order to understand your software's performance and behavior.
+
+* [Trace][trace_concept]: Trace can be thought of as a directed acyclic graph (DAG) of Spans, where the edges between Spans are defined as parent/child relationship.
```

Comment:
```suggestion
* [Trace][trace_concept]: Trace refers to distributed tracing. It can be thought of as a directed acyclic graph (DAG) of Spans, where the edges between Spans are defined as parent/child relationship.
```

Reply:
I would add a link to somewhere that explains distributed tracing.

Reply:
As well, "Span" should be a key concept.